Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -145,10 +147,19 @@ public NodejsProjectNode(NodejsProjectPackage package)
         }
 
         private void TryToAcquireTypings(IEnumerable<string> packages) {
+            bool isNewTypingsFolder = !Directory.Exists(Path.Combine(this.ProjectHome, "typings"));
             if (ShouldAcquireTypingsAutomatically && TypingsAcquirer != null) {
                 TypingsAcquirer
                     .AcquireTypings(packages, null /*redirector*/)
-                    .ContinueWith(x => x);
+                    .ContinueWith(x => {
+                        if (NodejsPackage.Instance.IntellisenseOptionsPage.ShowTypingsInfoBar &&
```

Comment:
Yep, that's a great idea - at least for when internet connection is lacking or something like that. If we do it lack of availability, i'd be concerned with how frequently it will fail to acquire typings for a newly installed package. We should chat with @bowdenk7, who's been delving into Salsa UX in general.